### DISCUSSION TITLE
Welcome to TeachersPetBot Discussions!

### DISCUSSION THREAD

**Discussion by Ashwinshankar98** [Ashwinshankar98/TeachersPetBot#21] [Ideas]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Ashwinshankar98** [comment]
just to give a brief  Intro on things that have been added in this iteration in the past few weeks are as  follows
* poll command
* Ping command
* Stats Command
* Attendance Command
* Extensive error handling in office hours, event creation, and database management module.

**Kindly share your opinions and enhancement that can be added to Phase2 of this project.**

**Reply by kailash1998s** [reply]
Some other functionalities added
- action on member join
- action on member quit/removal
- getInstructor command
- removeInstructor

**Comment by shakthinandana** [comment]
Database changes if any share here
Adding 'qna stored to db' as improvement to avoid repetition

**Comment by ithaaswin** [comment]
Added the attendance command
Introduced the custom help command

**Comment by ithaaswin** [comment]
Bot has been hosted using Heroku as discussed.
@Ashwinshankar98 has hosted using the current git repo

**Comment by ithaaswin** [comment]
Installing and testing guide has been updated by @ithaaswin and @Ashwinshankar98 
The issue can be closed now

**Reply by Ashwinshankar98** [reply]
verified and Closed.

**Comment by ithaaswin** [comment]
Readme updates has been provided by everyone and I think it's good to be closed now.

**Reply by Ashwinshankar98** [reply]
verified and Closed.

**Comment by ithaaswin** [comment]
As discussed in the meeting, video has been made and shared in our private channel
@kailash1998s will update the link in the repo once approved by everyone

**Comment by ithaaswin** [comment]
Fixed the bug where the main bot and testing bots are appearing in the absentees list.
It has been unit tested and is being closed.

**Comment by ithaaswin** [comment]
Workflow has been setup successfully by @Ashwinshankar98 
It is verified and issue is closed now.

**Comment by ithaaswin** [comment]
Test cases for attendance and help commands has been merged with the main
Code Coverage is upto the mark and test issue is closed.

**Reply by Ashwinshankar98** [reply]
verified

**Comment by saikaushik1997** [comment]
Added reminder notifications for exams and assignments that are coming up.

**Comment by saikaushik1997** [comment]
Handled the no close calls use case in the reminder notifications module.

**Comment by saikaushik1997** [comment]
Added functionality to add words to the existing profanity filter.

**Reply by saikaushik1997** [reply]
Fixed inconsistent output error between the custom profanity filter and the existing profanity filter.
